data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/unapproachable_east/una_spells.lst

```diff
@@ -4,12 +4,12 @@
-Beltyn's Burning Blood								TYPE:Arcane			CLASSES:Nar Demonbinder=4|Sorcerer,Wizard=5				SCHOOL:Necromancy											COMPS:V, S, M	CASTTIME:1 standard action	RANGE:Medium			TARGETAREA:One living creature											DURATION:(CASTERLEVEL) rounds (D)	SAVEINFO:Fortitude partial (see text)			SPELLRES:Yes				SOURCEPAGE:p.48	DESC:You deal 1d8 pts of acid and 1d8 pts of fire damage per round.
+Beltyn's Burning Blood								TYPE:Arcane			CLASSES:Sorcerer,Wizard=5				SCHOOL:Necromancy											COMPS:V, S, M	CASTTIME:1 standard action	RANGE:Medium			TARGETAREA:One living creature											DURATION:(CASTERLEVEL) rounds (D)	SAVEINFO:Fortitude partial (see text)			SPELLRES:Yes				SOURCEPAGE:p.48	DESC:You deal 1d8 pts of acid and 1d8 pts of fire damage per round.
 Bladebane										TYPE:Arcane.Divine	CLASSES:Paladin=2|Cleric=3|Sorcerer,Wizard=4				SCHOOL:Transmutation										COMPS:V, S, M	CASTTIME:1 standard action	RANGE:Touch				TARGETAREA:Weapon touched												DURATION:(CASTERLEVEL) rounds		SAVEINFO:Will negates (harmless, object)			SPELLRES:Yes (harmless, object)	SOURCEPAGE:p.48	DESC:Add +2 enhancement and +2d6 pts of damage to slashing weapon against a creature type of your choice.		TEMPBONUS:PC|WEAPON|TOHIT,DAMAGE|2
 # pre 6.2 had preapply 	BONUS:WEAPON|TOHIT,DAMAGE|2|PREAPPLY:Weapon,Slashing|TYPE=Enhancement.STACK
 Blink (Greater)				OUTPUTNAME:[NAME] Blink		TYPE:Arcane			CLASSES:Bard,Sorcerer,Wizard=5						SCHOOL:Transmutation										COMPS:V, S		CASTTIME:1 standard action	RANGE:Personal	ITEM:Potion	TARGETAREA:You														DURATION:(CASTERLEVEL) rounds (D)	SAVEINFO:None							SPELLRES:No					SOURCEPAGE:p.50	DESC:The caster vanishes and reappears for (CASTERLEVEL) rounds.
 Decastave										TYPE:Arcane			CLASSES:Sorcerer,Wizard=2							SCHOOL:Conjuration	SUBSCHOOL:Creation	DESCRIPTOR:Force			COMPS:V, S, M	CASTTIME:1 standard action	RANGE:0 ft.				TARGETAREA:Quarterstaff of force											DURATION:(CASTERLEVEL) rounds (D)	SAVEINFO:None							SPELLRES:Yes				SOURCEPAGE:p.49	DESC:Deals 1d6 damage per hit. Critical deals 1d8 bonus sonic damage.
-Fiendform										TYPE:Arcane			CLASSES:Nar Demonbinder,Sorcerer,Wizard=6					SCHOOL:Transmutation					DESCRIPTOR:Evil			COMPS:V, M		CASTTIME:1 action			RANGE:Personal	ITEM:Potion	TARGETAREA:The character												DURATION:(CASTERLEVEL) minutes	SAVEINFO:None							SPELLRES:No					SOURCEPAGE:p.49	DESC:Change into any fiendish creature, demon, or devil that can be summoned by summon monster I thru V.
+Fiendform										TYPE:Arcane			CLASSES:Sorcerer,Wizard=6					SCHOOL:Transmutation					DESCRIPTOR:Evil			COMPS:V, M		CASTTIME:1 action			RANGE:Personal	ITEM:Potion	TARGETAREA:The character												DURATION:(CASTERLEVEL) minutes	SAVEINFO:None							SPELLRES:No					SOURCEPAGE:p.49	DESC:Change into any fiendish creature, demon, or devil that can be summoned by summon monster I thru V.
```

Comment:
The class specification in the spell's declaration has been _moved_ to the .MOD list at the end of the spells.lst. Like Beltyn's Burning Blood (renamed simply, Burning Blood in the Spell Compendium), I make the spell assignments in the long .MOD list at the end of the spells.lst file. This centralized the assignment of the new (UNA book) spells with the assignment of the existing (SRD) spells for the prestige class.